app/src/main/java/pl/futurecollars/invoicing/model/Company.java

```diff
@@ -0,0 +1,19 @@
+package pl.futurecollars.invoicing.model;
+
+import lombok.Data;
+import lombok.Generated;
+
+@Data
+@Generated
```

Comment:
Nie chcemy sami używać adnotacji generated w kodzie, to jest raczej oznaczenie dla kompilatora stosowane przez biblioteki. Użyłeś tego pewnie, żeby pozbyć się problemu z jacoco, tutaj masz opisane rozwiązanie https://stackoverflow.com/questions/29520912/jacoco-exclude-generated-methods-using-it-with-lombok. Stwórz sobie plik lombok.config i dodaj co potrzeba :)